internal/commands/bundle.go

```diff
@@ -3,91 +3,22 @@ package commands
 import (
 	"bytes"
 	"context"
-	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"os"
-
 	"github.com/deislabs/cnab-go/bundle"
 	"github.com/docker/app/internal/packager"
 	"github.com/docker/app/internal/store"
 	"github.com/docker/app/types"
 	"github.com/docker/app/types/metadata"
-	"github.com/docker/cli/cli"
 	"github.com/docker/cli/cli/command"
 	"github.com/docker/cli/cli/config"
 	"github.com/docker/distribution/reference"
 	dockertypes "github.com/docker/docker/api/types"
 	"github.com/docker/docker/pkg/jsonmessage"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
-	"github.com/spf13/cobra"
+	"io/ioutil"
 )
 
-type bundleOptions struct {
```

Comment:
If it's not a command anymore, can we move it outside the package?

Reply:
:+1: